README.md

```diff
@@ -1,72 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# HTML/CSS & JavaScript capstone project - Conference Page
 
+> This project is a final capstone project for HTML/CSS & JavaScript. In this capstone project I built an online website for a conference using Cindy's design on Behance, but I personalized the content.  
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS & JAVASCRIPT.
 
-## Live Demo (if available)
+## Live Demo    
 
-[Live Demo Link](https://livedemo.com)
+[Live Site Demo](https://demix007.github.io/Capstone-Project-Style-Conference/)
 
+[Project Description Video](https://www.loom.com/share/afa22ca17b604af38c8cc5b72b1f09f9)
```

Comment:
- [ ] Nice working on your video presentation, like the previous reviewer mentioned the video is not loading, kindly fix this.